receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -1037,6 +1079,302 @@ func TestTranslateV2(t *testing.T) {
 				sm.Scope().SetName("scope1")
 				sm.Scope().SetVersion("v1")
 
+				return metrics
+			}(),
+		},
+		{
+			name: "CBNH translation",
+			request: &writev2.Request{
+				Symbols: []string{
+					"",
+					"__name__",
+					"test_cbnh_histogram",
+					"job",
+					"test",
+					"instance",
+					"localhost:8080",
+					"seconds",
+					"Test CBNH histogram",
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						LabelsRefs:       []uint32{1, 2, 3, 4, 5, 6}, // __name__=test_cbnh_histogram, job=test, instance=localhost:8080
+						CreatedTimestamp: 123456000,
+						Metadata: writev2.Metadata{
+							Type: writev2.Metadata_METRIC_TYPE_HISTOGRAM,
+						},
+						Histograms: []writev2.Histogram{
+							{
+								Timestamp:    123456789,
+								Schema:       -53, // CBNH schema
+								Sum:          100.5,
+								Count:        &writev2.Histogram_CountInt{CountInt: 50},
```

Comment:
PositiveDeltas are delta encoded, so the bucket values are 10, 25, 45, 50, 50 which adds up to 180.

```suggestion
								Count:        &writev2.Histogram_CountInt{CountInt: 180},
```